libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -218,8 +217,9 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
   }
 
   constructor(
-    private calendarHelper: CalendarHelper,
-    @Inject(LOCALE_ID) locale: string
+    @Inject(LOCALE_ID) locale: string,
+    private elementRef: ElementRef,
+    private cdr: ChangeDetectorRef
```

Comment:
If we are handling changedetection ourselves, maybe reflect this in the component metadata, i.e. `changeDetection: ChangeDetectionStrategy.OnPush` 🤔